TIL/topics/dart_basics/AbstractClass.md

```
@@ -0,0 +1,42 @@
+      ```
+    - 상속 클래스에 필수로 구현헤줘야하는 메서드를 SubClass에서 필수로 구현해줘야하는데 Override 받지 않거나 잘못된 메서드명으로 구현하는 경우가 있다.
+      ```dart
+      class Character {
+        // SubClass에서 필수로 구현해줘야한다.
+        void run() {}
+      }
+      class Person extends Character {
+        void ran() {} // run을 구현해야하는데 메서드명을 잘못 입력하는 문제가 있다.
+      }
+      ```
+  - ### 추상 클래스를 사용하기 위해서는 abstract keyword를 사용한다.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)